README.rst

```diff
@@ -52,8 +52,8 @@ scikit-learn requires:
 - SciPy (>= 0.17.0)
 - joblib (>= 0.11)
 
-**Scikit-learn 0.20 was the last version to support Python2.7.**
-Scikit-learn 0.21 and later require Python 3.5 or newer.
+**scikit-learn 0.20 was the last version to support Python 2.7.**
```

Comment:
We are far from consistent in whether we use capitalisation for scikit-learn particularly when at the beginning of a sentence

Reply:
ok gotcha